### PR TITLE
Refactor namespace to ai.ancf

### PR DESCRIPTION
**Pull request by kaikreuzer**

### CODE REVIEW

**Comment by patwlan** [comment]
We currently publish the libraries to maven central. We need to clarify, how to publish to the new namespace before we can change the group name.

File: build.gradle.kts [hunk 1]
```diff
@@ -20,7 +20,7 @@ plugins {
 }
 
 subprojects {
-    group = "io.github.lmos-ai.arc"
+    group = "ai.ancf.lmos.arc"
```
Comment (RobWin) [inline build.gradle.kts:23]: Why not just ai.ancf.lmos?
And then have all artifacts under this group name?

Comment (kaikreuzer) [reply build.gradle.kts]: Sounds good to me! I've changed it.

**Comment by kaikreuzer** [comment]
I've rebased this PR.

**Comment by patwlan** [comment]
Ok, we now have new namespace added to our sonatype account. On Monday we can sync with @pschwager on signing and updating the libs.

**Pull request merged**